tools/contrib/fsurdat_modifier.py

```diff
@@ -0,0 +1,356 @@
+#! /usr/bin/env python
+"""
+|------------------------------------------------------------------|
+|---------------------  Instructions  -----------------------------|
+|------------------------------------------------------------------|
+
+Instructions for running on Cheyenne/Casper:
+
+load the following into your local environment
+    module load python
+    ncar_pylib
+
+-------------------------------------------------------------------
+To see available options:
+    ./fsurdat_modifier.py --help
+-------------------------------------------------------------------
+
+This script reads a surface dataset (fsurdat file) and outputs a
+modified copy of the same file.
+
+-------------------------------------------------------------------
+To run the script:
+    ./fsurdat_modifier.py
+ 
+To remove NPL (ncar_pylib) from your environment on Cheyenne/Casper:
+    deactivate
+-------------------------------------------------------------------
+
+"""
+
+#  Import libraries
+from __future__ import print_function
+
+import sys
+import os
+import string
+import logging
+import subprocess
+import argparse
+
+import xarray as xr
+
+from datetime import date
+from getpass import getuser
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+myname = getuser()
+
+def get_parser():                                                                                                                                                                   
+        """
+        Get parser object for this script.
+
+        Command-line inputs
+        - fsurdat_in: input file (str)
+        - fsurdat_out: output file (str)
+        - variable: variable to modify (str)
+        - value: value assigned to the variable to be modified (float)
+        """
+        parser = ArgumentParser(description=__doc__,
+                               formatter_class=argparse.RawDescriptionHelpFormatter)
+
+        parser.print_usage = parser.print_help
+
+        parser.add_argument('--dom_pft', 
+                    help='Dominant PFT if overwrite_single_pft = .true. [default: %(default)s] ', 
+                    action="store", 
+                    dest="dom_pft",
+                    type =int,
+                    default=7)
+        parser.add_argument('--uni_snow', 
+                    help='Turn on the flag to create uniform snowpack. [default: %(default)s]', 
+                    action="store", 
+                    dest="uni_snow",
+                    default=False)
```

Comment:
- [ ] I feel like the typical thing to do for logicals like this (and others below) is to use action="store_true" (then you don't need default=False)